### ISSUE TITLE
It is possible to execute invalid query

### ISSUE THREAD

**Issue by svrcekmichal** [Youshido/GraphQL#43]
Query like this:
```
{
  user(id: 11457349) {

  } 
}
```
is invalid, but it is possible to execute it with package. It will resolve to all `user` keys, but I don't believe this is valid either. Can you confirm this? Or am I mistaken?

**Comment by viniychuk** [comment]
@svrcekmichal you're correct, it will be fixed & released this week.

**Comment by svrcekmichal** [comment]
@viniychuk ok great, I have also send you email about resolving, would be you be so good and give me some info about that? thanks once more

**Comment by viniychuk** [comment]
Fixed in 1.2.7

**Comment by svrcekmichal** [comment]
@viniychuk You can still fire nested empty query which is invalid
```
{
  user(id: 11457349) {
		photoAlbums {
      
    }
  }
}
```

Response:
```
{
  "data": {
    "user": {
      "photoAlbums": [
        [],
        [],
        [],
        []
      ]
    }
  }
}
```
